.github/workflows/release-python.yml

```diff
@@ -84,7 +87,7 @@ jobs:
 
   make_sdist:
     name: Make SDist
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Building on macOS instead of Ubuntu?

Reply:
This was an artifact from when Ubuntu wasn't working, but it doesn't matter either way.